AuthCC/Program.cs

```diff
@@ -30,31 +30,29 @@ public static async Task Main()
 
             (_configuration as ConfigurationRoot).Dispose();
 
-            using (var httpClient = new HttpClient())
+            using HttpClient httpClient = new ();
```

Comment:
Does ```using``` still need to be used here?

Reply:
yep, it's still a using statement, but the braces are optional now (starting with C# 8). You can just declare the using statement and the section of code it applies to is inferred. You still need the using keyword though because that's why initiates the proper automatic disposal.

From the [docs](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/using-statement):

> You can achieve the same result by putting the object inside a try block and then calling [Dispose](https://docs.microsoft.com/en-us/dotnet/api/system.idisposable.dispose) (or [DisposeAsync](https://docs.microsoft.com/en-us/dotnet/api/system.iasyncdisposable.disposeasync)) in a finally block; in fact, this is how the using statement **_and the using declaration_** are translated by the compiler.